### PR TITLE
Bug fixes

### PR DESCRIPTION
**Pull request by jatinkatyal13**
- CORS fix
- auth view bug on 400 req
- added test in travis

### CODE REVIEW

File: certificate_generator/settings.py [hunk 1]
```diff
@@ -162,3 +163,6 @@
 else:
     CORS_ORIGIN_WHITELIST = ALLOWED_HOSTS
 
+CORS_ALLOW_HEADERS = default_headers + (
```
Comment (jatinkatyal13) [inline certificate_generator/settings.py]: @aashutoshrathi this was the problem !

Comment (aashutoshrathi) [reply certificate_generator/settings.py]: :pray:

**Comment by aashutoshrathi** [comment]
Will you be adding tests in same PR?

**Comment by aashutoshrathi** [comment]
Needs a single line conflict resolve now

**Comment by jatinkatyal13** [comment]
Yup but I'll be needing your help in setting up tests !

File: .travis.yml [hunk 2]
```diff
@@ -23,3 +23,4 @@ before_script:
 script: 
   - python manage.py makemigrations api
   - python manage.py migrate
+  - pytest
```
Comment (aashutoshrathi) [inline .travis.yml]: What if we merge it without this line and create tests in another PR?

**Comment by aashutoshrathi** [comment]
ping @jatinkatyal13 
I think we should make this merge ready soon. :sweat_smile: 
We are very close to end.

File: .travis.yml (hunk 2 above)
Comment (jatinkatyal13) [reply .travis.yml]: Yup we should do this!
I'll revert the commit

Comment (aashutoshrathi) [reply .travis.yml]: Thanks :heart:

**Comment by jatinkatyal13** [comment]
@aashutoshrathi Now it should be good to go!

**Pull request merged**